app/components/emoji_keyboard.js

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {
+    View,
+    Text,
+    StyleSheet,
+    FlatList,
+    TouchableOpacity
+} from 'react-native';
+import {KeyboardRegistry} from 'react-native-keyboard-input';
+import {store} from 'app/mattermost';
```

Comment:
why do you need this here? this does not look right to me at all, the `Emoji` component does not need it